Source/WebKit/WebProcess/GPU/graphics/PrepareBackingStoreBuffersData.serialization.in

```diff
@@ -24,16 +24,26 @@
 
 header: "PrepareBackingStoreBuffersData.h"
 
-[CustomHeader, AdditionalEncoder=StreamConnectionEncoder] struct WebKit::PrepareBackingStoreBuffersInputData {
-    WebKit::BufferIdentifierSet bufferSet;
+[CustomHeader] struct WebKit::PrepareBackingStoreBuffersInputData {
```

Comment:
Now that you are importing a real existing concept, RemoteImageBufferSet, you should replace all references to the informal "BackingStoreBuffers" concept.

This could be called ImageBufferSetPrepareFrontBufferForDisplayInvocation with the exception that you come up with a better name.

See the toplevel comment for more.